websocket/polygon_test.go

```diff
@@ -90,6 +90,10 @@ func TestConnectAuthSuccess(t *testing.T) {
 	// closing before connecting shouldn't do anthing
 	c.Close()
 
+	// accessing output early shouldn't do anything
```

Comment:
I wish these were full sentences :)

Reply:
what specifically are you looking for? seems clear enough to me

Reply:
oh, that's not how i generally do inline comments. happy to change it if this is a polygon specific style but probably not in this PR.

Reply:
It's a style I promote. When I see comments like this I think things like "these comments don't matter because the writer didn't care enough to make them full sentences" or "these comments will be low quality because..." or "the writer didn't care about the reader needing to read it so I probably don't need to read it"

Reply:
hmm, i'm not sure i agree but i'll consider changing it in a future PR. for now, i don't want to update it in this one specific place when the rest of my inline comments match my style